index.js

```diff
@@ -72,9 +69,9 @@ module.exports = {
         return navis.activate(revision);
       },
 
-      fetchRevisions: function() {
+      fetchRevisions: function(context) {
         var navis = this.readConfig('navisDeploy');
-        this.log('Fetching revisions');
+        this.log('Fetching revisions from api.navis.io');
```

Comment:
I'm not sure `api.navis.io` will be clear to end users because it's kind of internal to us. Would just `Fetching revisions from Navis` make sense?